app/views/movies/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Movie Show Page</h1>
+
+<ol>
+  <li>Title: <%= @movie.title %></li>
+  <li>Creation Year: <%= @movie.creation_year %></li>
+  <li>Genre: <%= @movie.genre %></li>
+</ol>
+
+<h2>Actor Fast Facts</h2>
+<ol>
+  <li>Actors Youngest to Oldest:</li>
+  <ul>
+    <% @actors_sorted_by_age.each do |actor| %>
+      <li><%= actor.name %></li>
+    <% end %>
+  </ul>
+  <li>Average age of all actors: <%= @average_age_of_actors %></li>
+</ol>
+
+<h2>Add Actor</h2>
+<%= form_with(url: movie_path(@movie), data: { turbo: false }) do |form| %>
+  <%= form.label :actor_id, "Actor ID" %>
+  <%= form.text_field :actor_id %>
+  <%= form.submit "Add Actor" %>
+<% end %>
```

Comment:
Great use of HTML in this view.

The path for the form is not quite right though, I talked a little about it in the `routes.rb` file. The path for this form is going to be `"/movies/#{@movie.id}/actors"`, and the method is going to be `post`.

```ruby
<%= form_with(url: "/movies/#{@movie.id}/actors", method: :post, data: { turbo: false }) do |form| %>
  <%= form.label :actor_id, "Actor ID" %>
  <%= form.text_field :actor_id %>
  <%= form.submit "Add Actor" %>
<% end %>
```

The rest of the form is correct.